src/db/migrations/20230711034311_remove_temp_migration_col.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+
+export async function up(knex) {
+  if (true === await knex.schema.hasColumn('subscribers', 'db_migration_1')) {
```

Comment:
Not sure if we need the explicit `true === ` check (and ["yoda" condition](https://eslint.org/docs/latest/rules/yoda)), versus something like:

```suggestion
  if (await knex.schema.hasColumn('subscribers', 'db_migration_1')) {
```

There is also a [`table.dropColumns()`](https://knexjs.org/guide/schema-builder.html#dropcolumns) which accepts a variable number of column names, but not sure what happens if you try and delete non-existent columns.